modules/ppcp-settings/resources/js/Components/Screens/Onboarding/StepPaymentMethods.js

```diff
@@ -13,6 +13,11 @@ const StepPaymentMethods = ( {} ) => {
 		areOptionalPaymentMethodsEnabled,
 		setAreOptionalPaymentMethodsEnabled,
 	} = OnboardingHooks.useOptionalPaymentMethods();
+
+	const { storeCountry, storeCurrency } = CommonHooks.useWooSettings();
+
+    console.log(storeCountry, storeCurrency);
```

Comment:
leftover?

Reply:
That line is not even part of the latest branch... Strange that it shows up in the PR 🤔